data/resources.json

```diff
@@ -214,24 +207,20 @@
       "description": "Get notified about the important conference updates",
       "label": "Subscribe"
     },
-    "yourEmail": "Your email",
+    "yourEmail": "Seu email",
     "emailAddress": "Email Address",
     "firstName": "First Name",
     "lastName": "Last Name",
     "formTitle": "Get the latest updates, insights and news about GDG DevFest Ukraine sent to your inbox.",
     "emailRequired": "Please enter a valid email address.",
     "subscribe": "Subscribe",
     "subscribed": "Subscribed",
-    "close": "Close",
+    "close": "Fechado",
     "generalError": "Some error has occured. Please, try again later.",
     "toast": "Successfully subscribed!"
   },
-  "team": {
```

Comment:
you were not supposed to remove the existing keys.. I'll take the liberty to update your PR.

Reply:
Oh my bad... 

**Old key:value**
``` "team": "Organizadores" ``` 

**New key:value**

`"team": {
    "description": "Google is known all around the world. Everyone is 'googling', checking on 'maps' and communicating in 'gmail'. For simple users, they are services that just works, but not for us. Developers see much more: APIs, scalability issues, complex technology stacks. And that is what GDG is about.<br><br> [Google Developers Group (GDG) Lviv](http://lviv.gdg.org.ua/) - is open and volunteer geek community who create exciting projects and share experience about Google technologies with a passion.<br><br> Our goal is to organize space to connect the best industry experts with Ukrainian audience to boost development of IT."
  },`

So I thought it needed to change but I ended up removing it